tools/WinObjC.Packaging/ImportedByAnyDependent.targets

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="$(MSBuildProjectExtension) == '.nuproj'" Project="WinObjC.Packager.props" />
+  <Import Condition="$(MSBuildProjectExtension) == '.nuproj'" Project="WinObjC.Packager.targets" />
```

Comment:
><Import Condition="$(MSBuildProjectExtension) == '.nuproj'" Project="WinObjC.Packager.targets" /> [](start = 2, length = 97)

This line and line 7 could go inside their .props files instead. I'm fine with it as it is unless this way is wrong/not a best practice.